[2020-06-18 09:52:55,119] {taskinstance.py:669} INFO - Dependencies all met for <TaskInstance: slack_example.hellotask 2018-08-11T00:12:00+00:00 [queued]>
[2020-06-18 09:52:55,130] {taskinstance.py:669} INFO - Dependencies all met for <TaskInstance: slack_example.hellotask 2018-08-11T00:12:00+00:00 [queued]>
[2020-06-18 09:52:55,130] {taskinstance.py:879} INFO - 
--------------------------------------------------------------------------------
[2020-06-18 09:52:55,130] {taskinstance.py:880} INFO - Starting attempt 1 of 1
[2020-06-18 09:52:55,130] {taskinstance.py:881} INFO - 
--------------------------------------------------------------------------------
[2020-06-18 09:52:55,138] {taskinstance.py:900} INFO - Executing <Task(PythonOperator): hellotask> on 2018-08-11T00:12:00+00:00
[2020-06-18 09:52:55,140] {standard_task_runner.py:53} INFO - Started process 11080 to run task
[2020-06-18 09:52:55,201] {logging_mixin.py:112} INFO - Running %s on host %s <TaskInstance: slack_example.hellotask 2018-08-11T00:12:00+00:00 [running]> ip-172-31-76-14.ec2.internal
[2020-06-18 09:52:55,218] {python_operator.py:114} INFO - Done. Returned value was: Hello world!
[2020-06-18 09:52:55,229] {taskinstance.py:1065} INFO - Marking task as SUCCESS.dag_id=slack_example, task_id=hellotask, execution_date=20180811T001200, start_date=20200618T095255, end_date=20200618T095255
[2020-06-18 09:53:04,154] {logging_mixin.py:112} INFO - [2020-06-18 09:53:04,154] {local_task_job.py:78} ERROR - Received SIGTERM. Terminating subprocesses
